bugbot/rules/file_crash_bug.py

```diff
@@ -86,6 +86,18 @@ def get_bugs(self, date):
             "Firefox", "nightly"
         )
         signatures = data_fetcher.analyze()
+        # This is the last filtering stage which aims to avoid filling bugs for
+        # junky crashes, where the volume is low and the crashes do not show
+        # signals of being actionable or critical.
+        signatures = [
+            signature
+            for signature in signatures
+            if signature.num_installs > 20
+            or signature.num_crashes > 200
```

Comment:
Specific numerical thresholds could end up being too big or too small depending on channel etc. but I guess we're have to see how it goes.

Reply:
I agree. Currently, we file for nightly only. We are planning to expand that later.